src/Raven.Server/Documents/Queries/Results/QueryResultRetrieverBase.cs

```diff
@@ -408,47 +530,34 @@ private bool TryExtractValueFromIndex(ref RetrieverInput retrieverInput, FieldsT
 
                     return (null, results);
                 case BlittableJsonReaderObject nested:
-                    return (new Document
-                    {
-                        // https://issues.hibernatingrhinos.com/issue/RavenDB-19350
-                        // https://issues.hibernatingrhinos.com/issue/RavenDB-19127
-                        // We need to have a different instance of Id / LowerId, otherwise 
-                        // they will be disposed (but then try to be used)
-                        Id = doc.IgnoreDispose ? doc.Id?.CloneOnSameContext() : doc.Id,
-                        ChangeVector = doc.ChangeVector,
-                        Data = nested,
-                        Etag = doc.Etag,
-                        Flags = doc.Flags,
-                        LastModified = doc.LastModified,
-                        LowerId = doc.IgnoreDispose ? doc.LowerId?.CloneOnSameContext() : doc.LowerId,
-                        NonPersistentFlags = doc.NonPersistentFlags,
-                        StorageId = doc.StorageId,
-                        TransactionMarker = doc.TransactionMarker
-                    }, null);
-
+                {
+                    using var _ = doc;
```

Comment:
Sometimes you do `using var _ = doc;` sometimes 

```
using (doc)
{
}
```
Can we be consistent? (I prefer second syntax here because the scope is explicitly defined then)

Reply:
I still see this inconsistency. `using (doc)` is simpler and more explicit IMO